go-fuzz-build/main.go

```diff
@@ -325,25 +355,40 @@ func instrumentPackages(workdir string, deps map[string]bool, lits map[Literal]s
 
 	var ready []*Package
 	pkgs := make(map[string]*Package)
+	wg := sync.WaitGroup{}
+	mu := sync.Mutex{}
+	sem := make(chan bool, Par)
 	for pkg := range deps {
-		p := pkgs[pkg]
-		if p == nil {
-			p = &Package{name: pkg}
-			pkgs[pkg] = p
-		}
-		for _, imp := range goListList(pkg, "Imports") {
-			p1 := pkgs[imp]
-			if p1 == nil {
-				p1 = &Package{name: imp}
-				pkgs[imp] = p1
+		wg.Add(1)
+		sem <- true
+		go func(pkg string) {
+			defer wg.Done()
+			defer func() { <-sem }()
+
+			impList := goListList(pkg, "Imports")
```

Comment:
Here we only parallelize `go list`, right? So if we call `go list` once to extract all properties, this part will go away.